package.json

```
@@ -31,20 +27,114 @@
     "brute-force",
     "attack"
   ],
-  "devDependencies": {
-    "body-parser": "^1.20.0",
-    "eslint": "^7.15.0",
-    "eslint-config-prettier": "^7.0.0",
-    "eslint-plugin-prettier": "^3.2.0",
-    "express": "^4.17.1",
-    "husky": "^8.0.3",
-    "jest": "^29.0.3",
-    "prettier": "^2.2.1",
-    "pretty-quick": "^3.1.0",
-    "supertest": "^6.2.4"
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
+    }
+  },
+  "main": "dist/index.cjs",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/",
+    "tsconfig.json",
+    "readme.md",
+    "license.md",
+    "changelog.md"
+  ],
+  "engines": {
+    "node": ">= 16"
   },
   "scripts": {
-    "test": "eslint . && jest",
-    "precommit": "pretty-quick --staged"
+    "clean": "del-cli dist/ coverage/ *.log *.tmp *.bak *.tgz",
+    "build:cjs": "esbuild --platform=node --bundle --target=es2022 --format=cjs --outfile=dist/index.cjs --footer:js=\"module.exports = slowDown; module.exports.default = slowDown; module.exports.slowDown = slowDown; \" source/index.ts",
+    "build:esm": "esbuild --platform=node --bundle --target=es2022 --format=esm --outfile=dist/index.mjs source/index.ts",
+    "build:types": "dts-bundle-generator --out-file=dist/index.d.ts source/index.ts && cp dist/index.d.ts dist/index.d.cts && cp dist/index.d.ts dist/index.d.mts",
+    "compile": "run-s clean build:*",
+    "lint:code": "xo",
+    "lint:rest": "prettier --check .",
+    "lint": "run-s lint:*",
+    "format:code": "xo --fix",
+    "format:rest": "prettier --write .",
+    "format": "run-s format:*",
+    "test:lib": "jest test/library/",
+    "test:int": "jest test/integration/",
+    "test": "run-s lint test:*",
+    "pre-commit": "lint-staged",
+    "prepare": "run-s compile && husky install config/husky"
+  },
+  "peerDependencies": {
+    "express": ">= 4"
+  },
+  "dependencies": {
+    "express-rate-limit": "7.0.1"
```

We should loosen this up so that someone who wants to use both express-slow-down and express-rate-limit doesn't end up needing two installs of express-rate-limit.

Probably something like this:

```suggestion
    "express-rate-limit": ">= 7"
```

---

Or perhaps this if we don't want to risk express-rate-limit version 8+ introducing an incompatible change:

```suggestion
    "express-rate-limit": "^7.0.0"
```